roles/openshift_storage_nfs/tasks/main.yml

```diff
@@ -35,6 +35,9 @@
     - "{{ openshift.logging }}"
     - "{{ openshift.loggingops }}"
     - "{{ openshift.hosted.etcd }}"
+    - "{{ openshift.prometheus }}"
```

Comment:
The fact that this role knows about all possible things that could ever use NFS is a serious architectural coupling problem.

(zohar not picking on you, picking on where we need to change course in ansible)